.goreleaser.yml

```diff
@@ -13,13 +19,9 @@ builds:
       - -s -w -X github.com/odpf/siren/config.Version={{.Tag}} -X github.com/odpf/siren/config.BuildCommit={{.FullCommit}} -X github.com/odpf/siren/config.BuildDate={{.Date}}
     env:
       - CGO_ENABLED=0
-    goos:
-      - linux
-      - windows
-      - darwin
-    goarch:
-      - amd64
-      - 386
+    goos: [darwin, linux, windows]
+    goarch: [amd64, 386, arm, arm64]
```

Comment:
Remove arm for now not supported by https://github.com/prometheus/prometheus/tree/main/tsdb/fileutil